src/lib/EventProvider.js

```diff
@@ -10,13 +10,14 @@ class EventProvider {
     this._eventPublisher = eventPublisher;
   }
 
-  onNewEvent = (callback: (event: Event) => void) => {
+  onNewEvent = (callback: (event: Event) => void, eventNamePrefix: string = '*') => {
     this._eventPublisher.subscribe(
-      '*',
+      eventNamePrefix,
       this._onNewEvent(callback),
       {
         filterOptions: {
           listenToBroadcastedEvents: false,
+          listenToIPC: false,
```

Comment:
Also, I'm still not sure if we should filter the events for the general use case of eventProvider, but ok..let's keep it for now and change sometime later if somebody will need this.